backbone.js

```diff
@@ -1261,8 +1261,12 @@
 
     // Produces a DOM element to be assigned to your view. Exposed for
     // subclasses using an alternative DOM manipulation API.
-    _createElement: function(tagName) {
-      return document.createElement(tagName);
+    _createElement: function(tagName, namespace) {
+        if (typeof namespace === 'undefined') {
```

Comment:
Ok. Will make the change (I used typeof because I saw it elsewhere in backbone and I wanted to follow your standards.) 
By the way, the shortest way to get the real `undefined` is `void 0`, even when undefined is defined. This is what is done in underscore.